src/rulesets/oas3/__tests__/model-description.ts

```diff
@@ -1,58 +0,0 @@
-import { DiagnosticSeverity } from '@stoplight/types';
-import { RuleType, Spectral } from '../../../spectral';
-import * as ruleset from '../index.json';
-
-describe('components-schema-description', () => {
```

Comment:
Have we migrated this test? Cannot find it anywhere

Reply:
Dropped by @philsturgeon (as described in #725)

![image](https://user-images.githubusercontent.com/92363/69707898-e51d6a00-10fa-11ea-950a-7dc76dcd554e.png)

Reply:
Ah, right. Confused the names.